media/css/protocol/components/_sub-navigation.scss

```diff
@@ -13,12 +13,22 @@
         inset 0 10px 3px -10px rgba(29, 17, 51, 0.12);
 
     .mzp-l-content {
-        padding-top: 0;
-        padding-bottom: $spacing-md;
+        max-width: none;
```

Comment:
😬  https://github.com/mozilla/protocol/issues/933

Reply:
@stephaniehobson Exactly, i'll most likely abandon all the changes here and start with the most important fix straight upstream https://github.com/mozilla/protocol/pull/938